custom_components/tuya_local/devices/cleverio_ar10_ar20.yaml

```diff
@@ -0,0 +1,166 @@
+name: Cleverio AR10 / AR20
+products:
+  - id: urxauhwqufusih5h
+    name: Cleverio AR10 / AR20
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 5
+        max: 45
+      mapping:
+        - constraint: temperature_unit
+          conditions:
+            - dps_val: F
+              range:
+                min: 41
+                max: 113
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: low
+          value: eco
+        - dps_val: high
+          value: boost
+    - id: 12
+      name: fault_code
+      type: bitfield
+    - id: 101
+      name: hvac_action
+      type: boolean
+      mapping:
+        - constraint: hvac_mode
+          conditions:
+            - dps_val: false
+              value: "off"
+            - dps_val: true
+              mapping:
+                - dps_val: false
+                  value: idle
+                - dps_val: true
+                  value: heating
+    - id: 103
+      name: temperature_unit
+      type: string
+      mapping:
+        - dps_val: C
+          value: TEMP_CELSIUS
```

Comment:
These should just be C and F, so no mapping is needed (there is no recognition of the Python constant names from the source code, only the final values)

Reply:
Ah gotcha. I got the values from here: https://www.home-assistant.io/docs/configuration/basic/

For some reason the when I just had C and F the temperature would always display the current temperature in Celsius regardless of what I had selected. I realize now that Home Assistant will convert the temperature automatically based on my geolocation:
https://www.home-assistant.io/docs/configuration/basic/